### PR TITLE
Add notifications section for all events

### PR DESCRIPTION
**Pull request by priyesh777**
## Changes:
- Add notifications section to make a list of all events that occurred throughout the application

## This PR doesn't introduce any:
- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [ ] permission checks
- [ ] translations

### CODE REVIEW

File: src/components/Navbar/index.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React, { useContext, useCallback } from 'react';
 import { gql, useMutation } from '@apollo/client';
 import { _cs } from '@togglecorp/fujs';
 import { MdAdd } from 'react-icons/md';
+import { IoNotifications } from 'react-icons/io5';
```
Comment (tnagorra) [inline src/components/Navbar/index.tsx:5]: Let's use outline variant of the icons

File: src/components/Navbar/styles.css [hunk 2]
```diff
@@ -66,6 +66,11 @@
         .dropdown {
             font-size: var(--font-size-large);
         }
+
+        .notifications-link {
+            background: transparent;
```
Comment (tnagorra) [inline src/components/Navbar/styles.css:71]: Do we not have a transparent variant?

File: src/views/Notifiations/index.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import PageHeader from '#components/PageHeader';
+import Container from '#components/Container';
+
+import styles from './styles.css';
```
Comment (tnagorra) [inline src/views/Notifiations/index.tsx:7]: Please check these issues from linters.

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/components/Navbar/index.tsx (hunk 1 above)
Comment (priyesh777) [reply src/components/Navbar/index.tsx:5]: I did tried to use the outline variant but it didn't sync with the navbar components, due which I used the basic version

File: src/components/Navbar/styles.css (hunk 2 above)
Comment (priyesh777) [reply src/components/Navbar/styles.css:71]: The "ButtonLikeLink" is a custom component that uses "Link" as it's base component imported from react-router-dom, which doesn't have a transparent variant.

File: src/views/Notifiations/index.tsx (hunk 3 above)
Comment (priyesh777) [reply src/views/Notifiations/index.tsx:7]: The linters didn't show any issue in this code format.

File: src/components/Navbar/styles.css (hunk 2 above)
Comment (tnagorra) [reply src/components/Navbar/styles.css:71]: We do have a `transparent` field on ButtonLikeLink

File: src/views/Notifiations/NotificationCategory/index.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                className={styles.categoryName}
+                route={route.notifications}
+            >
+                Events
+            </SmartLink>
+            <SmartLink
+                className={styles.categoryName}
+                route={route.notifications}
+            >
+                Reports
```
Comment (tnagorra) [inline src/views/Notifiations/NotificationCategory/index.tsx:49]:
```suggestion
                Notifications
```

File: src/views/Notifiations/index.tsx (hunk 3 above)
Comment (tnagorra) [reply src/views/Notifiations/index.tsx:14]: Please check the logs on `yarn start` or `docker-compose up` in client

**Pull request merged**